app/src/main/java/hifian/hintahaukka/GUI/StoreListFragment.java

```diff
@@ -101,29 +163,88 @@ private void createStoreManager() {
         }
     }
 
-    /**
-     * Fragment gets the coordinates from the MainActivity.
-     * In tests zero coordinates are used instead.
-     */
-    private void getCoordinates() {
-        if (isRunningInTestEnvironment) {
-            this.lat = 0.0;
-            this.lon = 0.0;
-        } else {
-            this.lat = ((MainActivity)getActivity()).getLat();
-            this.lon = ((MainActivity)getActivity()).getLon();
-        }
-    }
-
     /**
      * Sets the isRunningInTestEnvironment variable true if this fragment has been launched in an android test.
      * Method calls the Main Activity, which causes a ClassCastException in test environment.
      */
     private void checkIfIsRunningInTestEnvironment() {
         try {
-            this.isRunningInTestEnvironment = ((MainActivity)getActivity()).isDisabled();
+            this.isRunningInTestEnvironment = ((MainActivity) getActivity()).isDisabled();
         } catch (ClassCastException e) {
             this.isRunningInTestEnvironment = true;
         }
     }
+
+    /**
+     * Finds the location of the user and updates the store list with the information.
+     * In tests location is set to a particular shop in Åland islands, beloved by the
```

Comment:
Should be "hated by the developing team"

Reply:
I disagree